### ISSUE TITLE
CLI and install setup

### ISSUE THREAD

**Issue by michaelsilverstein** [ssilber/FocusGame#1]
Not sure how this game works at all, but would probably benefit from CLI interaction and pip installation with the addition of a `setup.py` file with appropriate settings.

**Comment by ssilber** [comment]
Yeah I agree I should probably put in some extra effort here to make it more user-friendly. Honestly just posted exactly what I had for my final project (and overall I think this was a lame and not very "portfolio" worthy final project). I think it could benefit from a GUI as it's pretty much unplayable in the command line-- it basically only satisfies all of the requirements from the rubric.  

Do you have an example of a project you implemented this with? Would the idea be you package it all up so you can pip install all the modules and run the game from the command line with a `setup.py` file?

**Comment by michaelsilverstein** [comment]
Lol, mostly just annoying you, but this sounds cool! And I do have an example.. I am finishing up a report for finals rn, I'll get back to you soon.
